frontend/appflowy_flutter/lib/plugins/ai_chat/presentation/message/ai_message_bubble.dart

```diff
@@ -345,6 +370,12 @@ class AIResponseActionBar extends StatelessWidget {
         isInHoverBar: showDecoration,
         onTap: () => onRegenerate?.call(),
       ),
+      SaveToPageButton(
+        textMessage: message as TextMessage,
+        isInHoverBar: showDecoration,
+        onOverrideVisibility: onOverrideVisibility,
+        onTap: () {},
```

Comment:
If onTap can be null, we should mark it as nullable.

Reply:
Right on. I will remove this field